intranethome/templates/intranethome/mail_aliases.html

```diff
@@ -19,75 +19,87 @@
-            </tbody>
-
-        {% else %}
-            <tbody>
-                <td>{{email.local}}</td>
-            </tbody>
-        {% endif %}
-          {% endfor %}
-    </table>
-    <br>
-      </div>
-      {% endfor %}
-
-    <script src="{% static 'intranethome/js/accordion.js' %}"></script>
+    {% if error %}
+	<p>
+	    We're having a problem with our systems.  Please <a href="{{ error.link_url }}">contact us</a> so that we can fix the problem.
+	</p>
+    {% else %}
+	<button class="btn btn-default" onclick="expandAll()">Expand All</button>
+	<button class="btn btn-default" onclick="contractAll()">Contract All</button>
+
+	<h3>
+	  <a href="/mailaliases/">All</a> &nbsp; 
+	  <a href="/mailaliases/a/">A</a> &nbsp; 
+	  <a href="/mailaliases/b/">B</a> &nbsp; 
+	  <a href="/mailaliases/c/">C</a> &nbsp; 
+	  <a href="/mailaliases/d/">D</a> &nbsp; 
+	  <a href="/mailaliases/e/">E</a> &nbsp; 
+	  <a href="/mailaliases/f/">F</a> &nbsp; 
+	  <a href="/mailaliases/g/">G</a> &nbsp; 
+	  <a href="/mailaliases/h/">H</a> &nbsp; 
+	  <a href="/mailaliases/i/">I</a> &nbsp; 
+	  <a href="/mailaliases/j/">J</a> &nbsp; 
+	  <a href="/mailaliases/k/">K</a> &nbsp; 
+	  <a href="/mailaliases/l/">L</a> &nbsp; 
+	  <a href="/mailaliases/m/">M</a> &nbsp; 
+	  <a href="/mailaliases/n/">N</a> &nbsp; 
+	  <a href="/mailaliases/o/">O</a> &nbsp; 
+	  <a href="/mailaliases/p/">P</a> &nbsp; 
+	  <a href="/mailaliases/q/">Q</a> &nbsp; 
+	  <a href="/mailaliases/r/">R</a> &nbsp; 
+	  <a href="/mailaliases/s/">S</a> &nbsp; 
+	  <a href="/mailaliases/t/">T</a> &nbsp; 
+	  <a href="/mailaliases/u/">U</a> &nbsp; 
+	  <a href="/mailaliases/v/">V</a> &nbsp; 
+	  <a href="/mailaliases/w/">W</a> &nbsp; 
+	  <a href="/mailaliases/x/">X</a> &nbsp; 
+	  <a href="/mailaliases/y/">Y</a> &nbsp; 
+	  <a href="/mailaliases/z/">Z</a> &nbsp; 
+	  <a href="/mailaliases/number/">#</a>
+	</h3>
+
+	  <br>
+	  {% for alias, emails in final_data.items %}
+	  <button class="accordion">{{alias}}</button>
+	  <div class="panel">
+	<br>
+	<table class="table">
+	    {% for email in emails %}
+	    {% if email.triangle_brackets %}
+		<tbody>
+		    <td><a href="mailto:{{email.triangle_brackets.1}}">{{email.triangle_brackets.0}}</a></td>
+		    <td>{{email.triangle_brackets.2}}</td>
+		</tbody>
+
+	    {% elif email.parentheses %}
+		<tbody>
+		    <td><a href="mailto:{{email.parentheses.1}}">{{email.parentheses.0}}</a></td>
+		    <td>{{email.parentheses.2}}</td>
+		</tbody>
+
+	    {% elif email.plain_email %}
+		<tbody>
+		    <td><a href="mailto:{{email.plain_email}}">{{email.plain_email}}</a></td>
+		</tbody>
+
+	    {% elif email.note %}
+		<tbody>
+		    <td>{{email.note}}</td>
+		</tbody>
+
+	    {% else %}
+		<tbody>
+		    <td>{{email.local}}</td>
+		</tbody>
+	    {% endif %}
+	      {% endfor %}
+	</table>
+	<br>
+	  </div>
+	  {% endfor %}
+
+	<script src="{% static 'intranethome/js/accordion.js' %}"></script>
+
+    {% endif %}
```

Comment:
This template looks good but it could benefit from running `djhtml` from within the Vagrant environment:
```
djhtml -i intranethome/templates/intranethome/mail_aliases.html
```